### PR TITLE
commit unpacked libthumbor-1.3.2 for review

### PR DESCRIPTION
**Pull request by filippog**

### CODE REVIEW

File: python-libthumbor/libthumbor-1.3.2/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+libthumbor (1.3.2-1) UNRELEASED; urgency=medium
+
+  * Initial release.
```
Comment (filippog) [inline python-libthumbor/libthumbor-1.3.2/debian/changelog]: also `Closes: #783577`

File: python-libthumbor/libthumbor-1.3.2/debian/control [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Source: libthumbor
+Maintainer: Gilles Dubuc <gilles@wikimedia.org>
```
Comment (filippog) [inline python-libthumbor/libthumbor-1.3.2/debian/control]: consider having `Debian Python Modules Team <python-modules-team@lists.alioth.debian.org>` in `Uploaders` or `Maintainer`, see also http://python-modules.alioth.debian.org/policy.html#maintainership

File: python-libthumbor/libthumbor-1.3.2/debian/control [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+               python3-flake8,
+               python3-mock,
+               python3-nose,
+               python3-preggy,
+               python3-setuptools,
+               python3-six
+X-Python-Version: >= 2.6
+X-Python3-Version: >= 3.2
+Homepage: https://github.com/thumbor/libthumbor
+Vcs-Git: git://github.com/thumbor/libthumbor.git
```
Comment (filippog) [inline python-libthumbor/libthumbor-1.3.2/debian/control]: vcs-git and vcs-browser should refer to their debian location, IOW

```
Vcs-Git: https://anonscm.debian.org/git/python-modules/packages/libthumbor.git
Vcs-Browser: https://anonscm.debian.org/cgit/python-modules/packages/libthumbor.git
```

**Comment by filippog** [comment]
see comments inline, LGTM overall

**Comment by gi11es** [comment]
All concerns should be addressed in 6bb8741ca94148d9478643f2ab61bdfbb4b3bd00 Please check

**Pull request closed without merging**